main.js

```diff
@@ -0,0 +1,50 @@
+// Check if there are any existing books in local storage
+let books = JSON.parse(localStorage.getItem('books')) || [];
+// Function to remove a book from the collection
+function removeBook(book) {
+  books = books.filter((b) => b !== book);
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+// Function to display all books in the collection
+function displayBooks() {
+  const displayedBooks = document.getElementById('displayedBooks');
+  displayedBooks.innerHTML = '';
+
+  books.forEach((book) => {
+    const div = document.createElement('div');
+    div.innerHTML = `
+      <p>${book.title}</p>
+      <p>${book.author}</p>
+      <button class="removeBtn">Remove</button>
+      <hr>
+    `;
+
+    const removeBtn = div.querySelector('.removeBtn');
+    removeBtn.addEventListener('click', () => {
+      removeBook(book);
+      div.remove();
+    });
+
+    displayedBooks.appendChild(div);
+  });
+}
+
+// Function to add a new book to the collection
+function addBook(title, author) {
+  const newBook = { title, author };
+  books.push(newBook);
```

Comment:
- [ ] [Optional] Well done! You have a nice clean code. However, I have a suggestion to improve it. I recommend using `Array.concat()` instead of `Array.push()`. This change will provide more flexibility and maintain the immutability of the original arrays, which can be advantageous in this context. You can learn more about `Array.concat()` from the documentation [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat). Keep up the good work!